markdown/1-Introduction/exercises/ex_1/answers/samplefile.md

```diff
@@ -0,0 +1 @@
+Solution sample for exssfv.
```

Comment:
We cannot include any 'sample' files in the repository. Remove this, and also the one below (sdsdv.md)